cpmpy/solvers/pysat.py

```diff
@@ -93,12 +90,25 @@ def supported():
             # while we need the 'python-sat' package, some more checks:
             from pysat.formula import IDPool
             from pysat.solvers import Solver
+
+            # try to import pypblib once
+            if CPM_pysat._pb is None:
+                from pysat import card
+                CPM_pysat._card = card  # native
+                try:
+                    from pysat import pb  # require pypblib
+                    CPM_pysat._pb = pb
+                except (ModuleNotFoundError, NameError):  # pysat returns the wrong error type (latter i/o former)
+                    CPM_pysat._pb = False  # not installed, avoid reimporting
+
             return True
         except ModuleNotFoundError:
             return False
         except Exception as e:
             raise e
 
+    _pb = None  # holds pysat.pb module if its dependency `pypblib` installed
```

Comment:
I don't think this is needed, just like you don't do it for _card either... I think thats fine
(note how it is None here... just remove this?)

Reply:
I'll respond in the other `False`  v `None` comment